frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -351,6 +351,15 @@
                         success: function (data) {
                             if (data.result == 'SUCCESS') {
                                 componentsTable.row('#' + id).remove().draw(false);
+                                if (${componentList.size() == totalRows}) {
```

Comment:
Does this work? Just by reading code, I have doubts. `totalRows` is not a JS variable, so I wouldn't expect it to be available here. And if it were, this would only work once, because the second time around, the componentList would already be shorter than the initial total number of rows. 
I think what you'd rather want to use is `$('#componentCounter', '#componentCounter1', #'componentCounter2').each(function (counter){ counter.text(counter.text()-1);});`

Reply:
It works for me in testing (including for doing it more than once), but I can do it the way you suggest?

Reply:
OK, after a closer look, I understand why it works. I was a little confused.
Nevertheless, I think the code looks cleaner the way I suggested. Otherwise, you have this weird static `if (true)` or `if (false)` in the generated JS. Also, the actual decrement logig is repeated three times.